docs/src/pages/[platform]/getting-started/migration/migration.angular.mdx

```diff
@@ -215,3 +215,7 @@ please see the [Form Field Customization](../connected-components/authenticator/
 ### CSS Styling
 
 The 2.x version of the `Authenticator` has a completely different set of CSS variables. Please look over the [Amplify CSS Variables](../connected-components/authenticator/customization#styling) section for more information.
+
+### `onAuthUiStateChange`
+
+For previous use of `onAuthUiStateChange`, see [useAuthenticator](/angular/connected-components/authenticator/headless#authenticator-service).
```

Comment:
Does this apply to Angular?

Reply:
Oh yes, it should be `AuthenticatorService`, not `useAuthenticator`. Angular has services instead of hooks.

Reply:
Done d732e33c36d7f3acc899441f606c2b7ea3e533cd